files/scripts/sonic-netns-exec

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Wrapper to execute any command in a specific
+# network namespace.
+# Usage:
+# sonic-netns-exec <namespace instance #> <command to be executed>
```

Comment:
>sonic-netns-exec [](start = 2, length = 16)

To mimic the syntax of `ip netns`, I suggest call this script `sonic-netns`, and treat `exec` as the first parameter.

Usage as `sonic-netns exec <namespace> <command to be executed>`

Reply:
Changed the command to take namespace string as a parameter.
sonic-netns-exec "<namespace string>" <command to be executed>.
Empty command string will execute the command on host.
sonic-netns-exec "" <command to be executed>.